### DISCUSSION TITLE
Audio is broken! PLEASE HELPP!

### DISCUSSION THREAD

**Discussion by ThunderBirrddd** [mikebrady/shairport-sync#1904] [Q&A]
Hey, 
I am flabbergasted by the way Shairport-sync works, flawless airplay2 ah love it. 
Just took me some time to get it set up because I had to route my audio to a Bluetooth device which was a big hassle but got it working, now the issue I am facing is:

**The audio is constantly breaking as if someone is pausing and playing it again and again it plays okay for a second and then starts to lag like this, 
I have a video attached below. Let me know if it makes sense.**

Let me know if you require anything to troubleshoot.
Also, I am a literal beginner to Linux and have very little knowledge, 
Any help will be appreciated 
I am running ubuntu on a core 2 duo, 3gb ram machine.

https://github.com/user-attachments/assets/a4836dab-b1ed-4642-b968-e1847dfbc9e5

**Comment by mikebrady** [comment]
Thanks, but this should really be raised as an [issue](https://github.com/mikebrady/shairport-sync/issues/new/choose).

**Reply by ThunderBirrddd** [reply]
I had avoided the issue category because I thought the issue must be something from my end and not the software. 

Anyways I’ll change the category then.

**Reply by mikebrady** [reply]
Thanks — it should be easier to deal with as an issue.
